modules/gapi/test/common/gapi_video_tests_inl.hpp

```diff
@@ -7,7 +7,9 @@
 #ifndef OPENCV_GAPI_VIDEO_TESTS_INL_HPP
 #define OPENCV_GAPI_VIDEO_TESTS_INL_HPP
 
+#include <opencv2/gapi/cpu/video.hpp>
```

Comment:
It is not good to include a CPU-only header in tests common among different backends. If you have to use some backend-specific things (it's a structure here), we should consider some other solutions.
I'll try to find out

Reply:
Removed.